### PR TITLE
Add RubyGems Updates July 2022

### PR DESCRIPTION
**Pull request by lauragift21**
Draft update for July 2022.

Hi @deivid-rodriguez can you help review the updates from last month's work. thank you!

### CODE REVIEW

File: _posts/2022-08-08-july-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# RubyGems News
+
+This month in RubyGems, we released new versions of RubyGems [3.3.18](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3318--2022-07-14), [3.3.19](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3319--2022-07-27) and Bundler [2.3.18](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2318-july-14-2022), [2.3.19](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2319-july-27-2022).
+
+The following is a non-exhaustive list of other improvements included in the above releases (see the changelog for further information):
+ 
+- fixed an issue with Bundler on Windows that allows the new `x64-mingw-ucrt`, the default on **Ruby 3.1** to work seamlessly with the existing `platforms DSL` - [#5655](https://github.com/rubygems/rubygems/pull/5655). 
+- improved the  `bundler/setup` speed by lazily checking for missing platform-specific variants if nothing else has changed - [#5695](https://github.com/rubygems/rubygems/pull/5695).
```
Comment (deivid-rodriguez) [inline _posts/2022-08-08-july-rubygems-updates.md:17]: I would also include a reference to https://github.com/rubygems/rubygems/pull/5546 here, and don't go into details of each speed up. Just say: "Improve performance of `bundler/setup`".

File: _posts/2022-08-08-july-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- fixed some issues on `truffleruby` caused by other bug - [#5694](https://github.com/rubygems/rubygems/pull/5694), [#5746](https://github.com/rubygems/rubygems/pull/5746)
```
Comment (deivid-rodriguez) [inline _posts/2022-08-08-july-rubygems-updates.md:18]: I would just mention "Fixed several truffleruby issues", and include the links you mentioned plus https://github.com/rubygems/rubygems/pull/5711.

File: _posts/2022-08-08-july-rubygems-updates.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- fixed confusing permission error when copying compact index cache  - [#5709](https://github.com/rubygems/rubygems/pull/5709).
+- fixed an issue with Bundler printing the bug report template when it should give a better error and not suggest a bug - [#5726](https://github.com/rubygems/rubygems/pull/5726).
+- improved `gem not found` error messages to include expected source - [#5729](https://github.com/rubygems/rubygems/pull/5729).
+- merged a PR to fix `gem update --system` errors in some edge cases - [#5728](https://github.com/rubygems/rubygems/pull/5728), [#5737](https://github.com/rubygems/rubygems/pull/5737).
+- made RubyGems requiring `fileutils` lazier to avoid some flaky test failures - [#5738](https://github.com/rubygems/rubygems/pull/5738).
```
Comment (deivid-rodriguez) [inline _posts/2022-08-08-july-rubygems-updates.md:23]: I would probably spare the above entry (since it's unclear whether it was a real end user issue), and instead include links to the following three features (better at the top, before other entries)

```
- display mfa warnings on `gem signin`, to support work done on the rubygems.org side - [#5590](https://github.com/rubygems/rubygems/pull/5590).
- add the long requested feature of allowing to install gems from specific groups only - [#5579](https://github.com/rubygems/rubygems/pull/5759)
- extend `gem` DSL with a `force_ruby_platform` option - [#4049](https://github.com/rubygems/rubygems/pull/4049)
```

**Review by deivid-rodriguez — COMMENTED** [review]
I added some comments. We ship 3 features this month that seem worth mentioning!

File: _posts/2022-08-08-july-rubygems-updates.md (hunk 3 above)
Comment (lauragift21) [reply _posts/2022-08-08-july-rubygems-updates.md:23]: Thank you!! will add these entries to the post.

**Review by deivid-rodriguez — APPROVED** [review]
